Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,22 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string fName = GetInput("Enter your first name: ");
```

Comment:
Avoid using abreviations that are not a common knowledge.
fName should be renamed to firstName.